utils.py

```diff
@@ -431,13 +473,31 @@ def capitalize_string(input_string):
 
 
 def get_hex_color_for_category(category):
+    """Returns the category, it returns the color associated with the category,
+    if the category is present in the app constants else given a default color.
+
+    Args:
+        category : str. Category to get color
```

Comment:
Missing period(.) at the end. Also, please remove white space between category and colon(:).